docs/installation.md

```diff
@@ -9,20 +9,16 @@ different builds for different operating systems and accelerators.
 
 * **CPU-only (Linux/macOS/Windows)**
   ```
-  pip install -U "jax[cpu]"
+  pip install -U jax
```

Comment:
Perhaps we should leave a note somewhere about how to install versions older than 0.4.30.

Reply:
I left the `jax[cpu]` in the instructions for older releases.